stdlib/Dates/test/arithmetic.jl

```diff
@@ -500,7 +500,7 @@ end
 end
 
 @testset "Missing arithmetic" begin
-    for t ∈ [Date; Time; subtypes(DatePeriod); subtypes(TimePeriod)]
+    for t ∈ [Date Time Day Month Week Year Hour Microsecond Millisecond Minute Nanosecond Second]
```

Comment:
It is more natural to use a vector rather than a matrix even if the result is the same:
```suggestion
    for t ∈ [Date, Time, Day, Month, Week, Year, Hour, Microsecond, Millisecond, Minute, Nanosecond, Second]
```